### PR TITLE
v2.2.0 release notes for ARM Architecture 

### PR DESCRIPTION
**Pull request by ioannisg**
Release notes for ARM architecture

Based on #21201.

### CODE REVIEW

File: doc/releases/release-notes-2.2.rst [hunk 1]
```diff
@@ -5,10 +5,11 @@
 Zephyr 2.2.0 (Working Draft)
 ############################
 
-We are pleased to announce the release of Zephyr kernel version 2.2.0.
+We are pleased to announce the release of Zephyr RTOS version 2.2.0.
 
 Major enhancements with this release include:
 
+* We added support for 64-bit ARMv8-A Architecture (Experimental).
```
Comment (carlocaione) [inline doc/releases/release-notes-2.2.rst]: Please, do not use the word `experimental`. It is not an experiment and really detrimental to the work done. We can use `preliminary support` or `initial support`.

File: doc/releases/release-notes-2.2.rst [hunk 2]
```diff
@@ -144,7 +145,12 @@ Architectures
 
 * ARM:
 
-  * Removed support for CC2650
+  * Added initial support for ARMv8-A 64-bit architecture (Experimental)
```
Comment (carlocaione) [inline doc/releases/release-notes-2.2.rst]: ditto

File: doc/releases/release-notes-2.2.rst (hunk 1 above)
Comment (ioannisg) [reply doc/releases/release-notes-2.2.rst]: "initial" is fine (actually required, so I will add it)

"experimental" is hat we have been adding in similar cases - does not mean "it is not working" - it gives the impressions that something is not yet to be fully trusted; we don't make promises.

Comment (carlocaione) [reply doc/releases/release-notes-2.2.rst]: ok

Comment (ioannisg) [reply doc/releases/release-notes-2.2.rst]: And thanks once more @carlocaione for adding the support for v8-A :)

Comment (carlocaione) [reply doc/releases/release-notes-2.2.rst]: np ;) kudos to all the contributors.

**Review by jhedberg — APPROVED** [review]

**Review by ioannisg — COMMENTED** [review]
@jhedberg I just re-arranged the bullet-points a bit, after the added commits by @stephanosio .
As far as I am concerned this is OK to push in.

**Review by carlocaione — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Pull request merged**